code/game/objects/items/devices/radio/encryptionkey.dm

```diff
@@ -11,16 +11,19 @@
 
 /obj/item/encryptionkey/Initialize(mapload)
 	. = ..()
-	if(!channels.len)
-		desc = "An encryption key for a radio headset.  Has no special codes in it. You should probably tell a coder!"
+	if(!channels.len && !translate_binary)
+		desc += " Has no special codes in it. You should probably tell a coder!"
 
 /obj/item/encryptionkey/examine(mob/user)
 	. = ..()
-	if(LAZYLEN(channels))
+	if(LAZYLEN(channels) || translate_binary)
 		var/list/examine_text_list = list()
 		for(var/i in channels)
 			examine_text_list += "[GLOB.channel_tokens[i]] - [lowertext(i)]"
 
+		if(translate_binary)
+			examine_text_list += "[GLOB.channel_tokens[MODE_BINARY]] - [MODE_BINARY]"
+
```

Comment:
god why is binary the worst

Reply:
I have no idea why binary can't be treated like a normal radio channel but it scares me

Reply:
It's supposed to be unblockable or something. I don't know why.

Reply:
Then why not treat it like the syndi channel? Seems like it'd be better

Reply:
guys radio code is so old it has a readme calling it new

Reply:
Syndicate channel is blockable, the machine is on the nukie ship